### ISSUE TITLE
update wayland 1.18.0 -> 1.19.0

### ISSUE THREAD

**Issue by d4g** [NixOS/nixpkgs#112864]
###### Checklist

- [x] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name: wayland

current version: 1.18.0
desired version: 1.19.0

###### Notify maintainers

maintainers:
@primeos 
@codyopel

###### Note for maintainers

Please tag this issue in your PR.

**Comment by d4g** [comment]
I tried it myself, but it seems that something with the applied patches goes wront?

```
building '/nix/store/g8wrdrlqfnpm36wx7dqawk73aax553iz-wayland-1.19.0.drv'...

trying https://github.com/swaywm/wlroots/archive/5e19e0053a5800252a27ce127fd015d641ee2e9e.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
unpacking sources
unpacking source archive /nix/store/f2xgdv25jkv2rd2i00z2c1iza3n0jac2-wayland-1.19.0.tar.xz
source root is wayland-1.19.0
setting SOURCE_DATE_EPOCH to timestamp 1611766152 of file wayland-1.19.0/src/wayland-version.h
patching sources
applying patch /nix/store/4m12lw220plcb5wnk86j6pjs9qp6rnlc-e53e0edf0f892670f3e8c5dd527b3bb22335d32d.patch
patching file doc/publican/meson.build
Reversed (or previously applied) patch detected!  Assume -R? [n]
Apply anyway? [n]
Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file doc/publican/meson.build.rej
builder for '/nix/store/g8wrdrlqfnpm36wx7dqawk73aax553iz-wayland-1.19.0.drv' failed with exit code 1
cannot build derivation '/nix/store/0p6ls1gpni8lnw8sz7w3wvaqpka7b690-wlroots-0.12.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/0p6ls1gpni8lnw8sz7w3wvaqpka7b690-wlroots-0.12.0.drv' failed

```

**Comment by r-burns** [comment]
The update to wayland 1.19 is in staging-next right now: https://github.com/NixOS/nixpkgs/commit/106ca78aff3395c5074dd3714f9c3a569469c842

**Comment by d4g** [comment]
Oh - sorry. I loocked for the PR but I didn't find it.

**Comment by d4g** [comment]
Solved via  #107324
